src/app/mini-games/sprint/sprint-mini-game.component.html

```diff
@@ -1 +1,79 @@
-<p>sprint-mini-game works!</p>
+<div class="background-container">
+  <div *ngIf="game.gameState === 'PLAY'" class="game__counter">
```

Comment:
@Suggestion
You have 'PLAY' in enum so it's better to use it

Reply:
Enum isn't possible to use in the template directly, so I created property in the component and used it:
component:
  GAME_STATE = {
    SETUP: GameState.SETUP,
    READY: GameState.READY,
    PLAY: GameState.PLAY,
    PAUSE: GameState.PAUSE,
    OVER: GameState.OVER,
  };
template: 
<div *ngIf="game.gameState === GAME_STATE.PLAY">

Or there is a better way to do it?

Reply:
Maybe it'll work:

``` public GameState = GameStateEnum;```